README.rst

```diff
@@ -132,9 +136,11 @@ If your CodeJail is properly configured to use safe_exec, try these
 commands at your Python terminal::
 
     import codejail.jail_code
-    codejail.jail_code.configure('python', '<SANDENV>/bin/python')
+    codejail.jail_code.configure('python', '<SANDENV>/bin/python', user='sandbox')
     import codejail.safe_exec
-    codejail.safe_exec.safe_exec("import os\nos.system('ls /etc')", {})
+    jailed_globals = {}
+    codejail.safe_exec.safe_exec("output=open('/etc/passwd').read()", jailed_globals)
+    jailed_globals
```

Comment:
Because it confirms that jailing failed. There are three possible outcomes: Noisy confinement from safe_exec (exception, error logging, etc.), quiet confinement (returns normally, jailed_globals unaltered), and quiet "jailbreak". Printing jailed_globals discriminates between the latter two.

Maybe I should change it to this:

```suggestion
    print(jailed_globals)  # should be unreachable if codejail is working properly
```